apps/accounts/migrations/0006_alter_user_about_alter_user_img_url.py

```
@@ -0,0 +1,23 @@
+    operations = [
+        migrations.AlterField(
+            model_name='user',
+            name='about',
+            field=models.TextField(blank=True, verbose_name='about'),
+        ),
+        migrations.AlterField(
+            model_name='user',
+            name='img_url',
+            field=models.URLField(blank=True, verbose_name='image url'),
+        ),
+    ]
```

Ruff (0.8.2)

Mutable class attributes should be annotated with `typing.ClassVar`

(RUF012)